Ginger/GingerCore/Drivers/Common/HTMLElementInfo.cs

```diff
@@ -16,12 +16,15 @@
 */
 #endregion
 
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.Repository;
 using Amdocs.Ginger.Common.UIElement;
+using Amdocs.Ginger.Repository;
 using HtmlAgilityPack;
 
 namespace GingerCore.Drivers.Common
 {
-    public class HTMLElementInfo : ElementInfo
+    public class HTMLElementInfo : ElementInfo, IParentOptionalValuesObject
     {       
```

Comment:
why we doing it to HTMLElementInfo and not to ElementInfo?

Reply:
Changed to ElementInfo